lib/internal/test_runner/runner.js

```diff
@@ -496,6 +496,9 @@ function watchFiles(testFiles, opts) {
     }
 
     await runningSubtests.get(file);
+
+    opts.root.reporter[kEmitMessage]('test:watch:restarted');
```

Comment:
Hey @OkunadeNaheem, it looks like this branch was created from a very outdated main branch.

This addition was already made two months ago in this PR: https://github.com/nodejs/node/pull/57903

Here's the line: https://github.com/nodejs/node/blob/main/lib/internal/test_runner/runner.js#L526